extensions/src/objectDetection/index.ts

```diff
@@ -0,0 +1,161 @@
+import { ArgumentType, BlockType, ExtensionMenuDisplayDetails, extension, block, untilTimePassed, rgbToHex, RGBObject } from "$common";
+import { ObjectDetector as ObjectDetectorClass } from "@mediapipe/tasks-vision"
```

Comment:
let's make this an `import type...` statement to make it clear we aren't using/bundling the underling code:

```ts
import type { ObjectDetector as ObjectDetectorClass } from "@mediapipe/tasks-vision"

```